backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5581070_sys_gh10766_ExternalSystem_Window_Improvements.sql

```diff
@@ -0,0 +1,439 @@
+-- 2021-03-03T14:27:38.464Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Index_Table (AD_Client_ID,AD_Index_Table_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,Description,EntityType,IsActive,IsUnique,Name,Processing,Updated,UpdatedBy) VALUES (0,540572,0,541577,TO_TIMESTAMP('2021-03-03 16:27:38','YYYY-MM-DD HH24:MI:SS'),100,'Unique constraint on parent config id','U','Y','Y','IDX_S_ExternalSystemAlberta_unique_parent_id','N',TO_TIMESTAMP('2021-03-03 16:27:38','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2021-03-03T14:27:38.471Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Index_Table_Trl (AD_Language,AD_Index_Table_ID, ErrorMsg, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Index_Table_ID, t.ErrorMsg, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Index_Table t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Index_Table_ID=540572 AND NOT EXISTS (SELECT 1 FROM AD_Index_Table_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Index_Table_ID=t.AD_Index_Table_ID)
+;
+
+-- 2021-03-03T14:27:45.638Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Index_Table SET EntityType='D',Updated=TO_TIMESTAMP('2021-03-03 16:27:45','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Index_Table_ID=540572
+;
+
+-- 2021-03-03T14:28:00.360Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Index_Column (AD_Client_ID,AD_Column_ID,AD_Index_Column_ID,AD_Index_Table_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,SeqNo,Updated,UpdatedBy) VALUES (0,572766,541064,540572,0,TO_TIMESTAMP('2021-03-03 16:28:00','YYYY-MM-DD HH24:MI:SS'),100,'D','Y',10,TO_TIMESTAMP('2021-03-03 16:28:00','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2021-03-03T14:28:53.749Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Index_Table (AD_Client_ID,AD_Index_Table_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,Description,EntityType,IsActive,IsUnique,Name,Processing,Updated,UpdatedBy) VALUES (0,540573,0,541585,TO_TIMESTAMP('2021-03-03 16:28:53','YYYY-MM-DD HH24:MI:SS'),100,'Unique constraint on external config parent id','D','Y','Y','IDX_S_ExternalSystemShopware6_unique_parent_id','N',TO_TIMESTAMP('2021-03-03 16:28:53','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
~~When creating a unique constraint, please always create it with whereclause "IsActive='Y'", so that a supporter has the option of deactivating it in order to get it out of the way and create a new one~~
in this case it's OK i think, as there will always be *one* record